Helidon v2.1.0

aberinnj opened the pull request:
- Dependencies still using JDBC v19 and other packages that may be uneccessary, **Updated**
- No NPE guards for missing environment variables, **Updated**
- Environment variables not homogenous, and undocumented, **Updated**
- Add README for running instructions, **Updated**
- Improve code to be more flexible, **Updated**

aberinnj committed:
Create serial-config.properties

aberinnj committed:
Update pom.xml

aberinnj committed:
Added appdatasource

aberinnj committed:
Added serial-config

aberinnj committed:
Updated database access to support other methods of connecting

aberinnj committed:
Updated manifest to support new environment variables

aberinnj committed:
Removing target files

aberinnj committed:
Sample SQL file

aberinnj committed:
Added README

aberinnj committed:
updated README

aberinnj committed:
Updated port

aberinnj committed:
Updated scripts to reflect new table

aberinnj commented:
@Kuassim Please review when possible and merge

Kuassim commented on mtdrworkshop/backend/src/main/java/com/oracle/database/AppDatasource.java:
```diff
@@ -0,0 +1,72 @@
+package com.oracle.database;
```
Can you please add the Oracle Copyrights

Kuassim commented on mtdrworkshop/backend/src/main/java/com/oracle/database/AppDatasource.java:
```diff
@@ -0,0 +1,72 @@
+
+    // connection pooling using UCP
+    private final PoolDataSource pds;
+    private static AppDatasource instance = null;
```
It looks like you have 3 spaces instead of 2; not a big deal as long as it is not a tab.

Kuassim reviewed (changes requested):
1. One Java code does not have Copyright (unless I am not seeing right).
2. Make sure to use 2 tabs for indentation at each level (it looks like you are using 3 and sometimes more than 3.
3. Use try-with-resource for code blocks with statements and resultSets; to  make sure these are closed/freed.

aberinnj committed:
Updated formatting and indentation

aberinnj committed:
Formatted file

aberinnj committed:
Updated indentation

aberinnj committed:
Updated Java files

aberinnj commented:
@Kuassim 
1. fixed
2. fixed I hope! Some code are using builder methods so they may be indented more
3. I do not see any files having this signature

aberinnj commented:
@jeandelavarene Please review and merge when possible
